src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -30,6 +32,14 @@ private static void setupPresetTests(Application application) {
 
 		application.addTest("Figure Joe 1", selectTestFigureOptionListener);
 		application.addTest("Figure Joe 2",actionEvent -> FiguresJoe.figureScript2(DriverFeature.getDriverManager().getCurrentDriver()));
+		application.addTest("Square",event -> {
+			ComplexCommand command = FigureFactory.square();
+			command.execute(DriverFeature.getDriverManager().getCurrentDriver());
+		});
+		application.addTest("Triangle",event -> {
+			ComplexCommand command = FigureFactory.triangle();
```

Comment:
It is good to keep a convention used in the original code (class instance instead of lambda)